docs/release-notes/cloud/cloud-2023-12-01-new-coverage-engine-status-reports.md

```diff
@@ -0,0 +1,73 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# New Coverage engine status checks December 1st, 2023<!-- TODO ALA-695 Update to launch date. Search and replace this. Don't forget to also update the file name -->
+
+!!! note
+    This release note applies if you are using Codacy to send code quality information to your Git provider (see how on [GitHub](../../repositories-configure/integrations/github-integration.md#status-checks), [GitLab](../../repositories-configure/integrations/gitlab-integration.md#pull-request-status), and [Bitbucket](../../repositories-configure/integrations/bitbucket-integration.md#pull-request-status)).
+
+On December 1st 2023 we activated the new, faster Coverage engine and set it to send coverage information to your Git provider.
+
+As a consequence, you can now see two additional checks on your pull requests, marked **beta**, alongside the preexisting Codacy Coverage check.
+
+![New Coverage status report](../images/ala-593.png)<!-- TODO ALA-695 Update screenshot and add for all providers -->
+
+This is part of an ongoing effort to improve the speed and <span class="skip-vale">actionability</span> of the insights provided by Codacy.
+
+## Deprecation and removal of the Coverage engine
+
+Currently, Codacy sends coverage information to your Git provider from both the old and new Coverage engines. We will deprecate and eventually retire the old engine on the following dates:
+
+<table>
+  <thead>
+    <th>Date</th>
+    <th>Event</th>
+    <th>What to expect</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>February 14th, 2024</td>
+      <td>Deprecation of the old Coverage status check</td>
+      <td>
+        <p>From this day on, the new checks will no longer be marked <strong>beta</strong> and will be considered the de facto status checks for Codacy Coverage. The old check will be marked <strong>deprecated</strong> but will keep working.</p>
+        <p>If on this date you are still using the old check to block merging pull requests, you should update your Git provider settings.</p>
```

Comment:
We could add a link to the instructions on how to **update your Git provider settings**.